yamlfmt.unwrap.sh

```diff
@@ -9,9 +9,9 @@ source hack/utils.sh
 
 while IFS= read -r file; do
   # Step 1: Unwrap quotes from specific fields
-  os::util::sed -E "s/([pP]ort|replicas|enabled|azCount): '\{\{\s*([^}]+)\s*\}\}'$/\1: {{ \2 }}/g" "$file"
+  os::util::sed -E "s/([pP]ort|replicas|enabled|azCount|expression): '\{\{\s*([^}]+)\s*\}\}'$/\1: {{ \2 }}/g" "$file"
```

Comment:
Makes sense, it's the smallest sin, of the ones we have available to us:

1. ignore the directory entirely
2. make some uber-complex loading & parsing thing for files that are not YAML but may become YAML after a template pass
3. add more sins to this list of sins